Dockerfile

```diff
@@ -19,7 +19,8 @@ RUN apk add --no-cache \
 
 # Download go dependencies
 WORKDIR /osmosis
-COPY go.mod go.sum ./
+COPY go.work go.mod go.sum ./
+COPY osmomath/ ./osmomath
```

Comment:
Is it possible to only copy `osmomath/go.mod` and `osmomath/go.sum` here?

This is so that updates to `osmomath` code do not trigger wget for wasvm. Having only `go.mod` will allow that layer to be cached

Reply:
Let me try this, but I believe the answer is no. This is because the main go.mod replaces it with a local folder path, rather than downloading this folder from a normal upstream mirror.

Though we could move all these copies until after the wasmvm wget, I don't understand why we do the go mod download before the wget